src/components/standup/standupContainer.module.scss

```diff
@@ -0,0 +1,152 @@
+$theme-focusHighlight: #1c64f2;
+$theme-primaryBackground: #1c325e;
+$theme-whiteColor: #ffffff;
+$theme-bannerFontColor: #f1416c;
+$theme-secondaryBackground: #1e429f;
+$theme-CeriseRed: #d61f69;
+$theme-headingColor: #6b7280;
+$theme-inputFieldBorder: #d1d5db;
+$theme-nonActiveButtonBackground: #e5e7eb;
+$radius: 0.5rem;
+
+.standupContainer {
+    display: flex;
+    flex-direction: column;
+    margin-top: 1.88rem;
+    width: 100%;
+    align-items: center;
+    justify-content: center;
+}
+
+.standupBanner {
+    background-color: $theme-primaryBackground;
+    width: 84%;
+    height: 12%;
+    color: $theme-whiteColor;
+    padding: 2rem;
+    border-radius: $radius;
+}
+
+.standupBanner .bannerPara {
+    font-family: 'Poppins', sans-serif;
+    font-size: 1.24rem;
+    font-weight: 600;
+}
+
+.standupBanner .bannerPara .totalMissedUpdate {
```

Comment:
```suggestion
.totalMissedUpdate {
```

Reply:
Can you set the style like this (without using parent class) ?

Reply:
fixed